v10.15/catalog-template.json

```diff
@@ -63,6 +73,10 @@
         {
             "schema": "olm.bundle",
             "image": "registry.redhat.io/openshift4-wincw/windows-machine-config-operator-bundle@sha256:452a7634363c3f39aacab727c5cf7eef2c5bd95c39b5babb40a70b962f4f97c3"
+        },
+       {
+            "schema": "olm.bundle",
+            "image": "quay.io/redhat-user-workloads/windows-machine-conf-tenant/windows-machine-config-operator/windows-machine-config-operator-bundle-release-4-15@sha256:8ca4e252fa170d92e4adc2603b220af7897cd6eb2ff9f808b05cc045e4ff1a9d"
```

Comment:
This should be a reference to `registry.redhat.io` not quay.
If this is generated from opm, there is an issue on how the WMCO bundle was built. 
The bundle needs to refer to `registry.redhat.io`, and during development we can use a IDMS to replace it with a quay URL.

Reply:
so we dont need to run this every time we update this file?
https://github.com/openshift/windows-machine-config-operator-fbc?tab=readme-ov-file#updating-a-patch-version-before-release